### ISSUE TITLE
Woocommerce 9.7.0 product TAB's accessibility issue

### ISSUE THREAD

**Issue by betayazilim** [woothemes/woocommerce#55980]
### Prerequisites

- [x] I have carried out troubleshooting steps and I believe I have found a bug.
- [x] I have searched for similar bugs in both open and closed issues and cannot find a duplicate.

### Describe the bug

Hi there,

Woocommerce 9.7.0 has accessibility issue with product tabs.
https://prnt.sc/Qj5YQ_vF9pFf

ARIA
Elements with an ARIA [role] that require children to contain a specific [role] are missing some or all of those required children.
[role]s are not contained by their required parent element
List items (<li>) are not contained within <ul>, <ol> or <menu> parent elements.

But when I downgrade 9.5.2 it is OK.
https://prnt.sc/oy0enoHVLD9J

fyi.

### Expected behavior

No accessibility issue for sure.

### Actual behavior

accessibility issue on product tabs.

### Steps to reproduce

1. pagespeed tool can show details

### WordPress Environment

Wordpress latest
php 8.3

### Isolating the problem

- [x] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [x] This bug happens with a default WordPress theme active, or [Storefront](https://woocommerce.com/products/storefront/).
- [x] I can reproduce this bug consistently using the steps above.

**Comment by gigitux** [comment]
Thanks for opening this issue! I'm able to replicate this issue. It looks like that the PR that introduced this regression is https://github.com/woocommerce/woocommerce/pull/53158. We will investigate it!

**Comment by Manussakis** [comment]
@ned-bs Thanks for opening this issue.
@gigitux I am looking into it.

**Comment by betayazilim** [comment]
It is not completed.  I still have this problem with latest versions of woocommerce

**Comment by Aaronomary1** [comment]
Still seeing this... what is wrong here? Please fix accessibility is critical...

**Comment by Sammiess** [comment]
I can confirm this is not completed as it is still an issue. When you remove the product tab and replace it with an accordion block the issue is resolved.

**Comment by gigitux** [comment]
Which version of WooCommerce are you running? This patch will land on WooCommerce 9.9 (not released yet)
